sqlmesh/core/model/definition.py

```diff
@@ -2018,7 +2026,22 @@ def _create_model(
         python_env=python_env,
     )
 
+    env: t.Dict[str, t.Any] = {}
+
+    for signal_name, _ in model.signals:
+        if signal_definitions and signal_name in signal_definitions:
+            build_env(
+                signal_definitions[signal_name].func, env=env, name=signal_name, path=module_path
+            )
+
+    # _, referenced_variables = parse_dependencies(env)
+
+    # variables = {k: v for k, v in (variables or {}).items() if k in referenced_variables}
```

Comment:
Why is this commented out?

Reply:
deleting it as it's unnecessary, it was a todo